pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -681,47 +742,53 @@ func (p *MasqueradePodInterface) createBridge() error {
 		return err
 	}
 
+	if Handler.IsIpv6Enabled(p.podNicLink) {
```

Comment:
`IsIPv6Enabled` seems to be called multiple times. Consider running the check when constructing the `MasqueratePodInterface` object and store the result in the struct.